web/rest/platform/src/Controller/Provider/TerminalModel/DefaultTemplateAction.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Controller\Provider\TerminalModel;
+
+use Ivoz\Provider\Domain\Model\TerminalModel\TerminalModelInterface;
+use Ivoz\Provider\Domain\Model\TerminalModel\TerminalModelRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class DefaultTemplateAction
+{
+    public function __construct(
+        private TerminalModelRepository $terminalModelRepository
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $id = (int) $request->attributes->get('id');
+        $type = (string) $request->query->get('type');
+        if (!in_array($type, ['generic', 'specific'])) {
+            throw new \InvalidArgumentException('Unknown type');
+        }
+
+        /** @var TerminalModelInterface | null $terminalModel */
+        $terminalModel = $this->terminalModelRepository->find($id);
+        if (!$terminalModel) {
+            throw new NotFoundHttpException('Terminal model not found');
+        }
+
+        $iden = $terminalModel->getIden();
+        $sourceFile = sprintf(
+            '/opt/irontec/ivozprovider/microservices/provision/templates/provisioning/%s/%s.cfg',
```

Comment:
Is not a better idea to set, this configuration under the service.yaml of microservice, similar to the config related to:

`bind:
            $storagePath: '/opt/irontec/ivozprovider/storage'`

`bind:
            $provisionPath: '/opt/irontec/ivozprovider/microservices/provision/templates/provisioning'`